Invalid use of a restricted identifier `yield`

Found another `yield` method

https://github.com/jbee/silk/blob/25426965253062d0238e2db97b19175f042892a7/src/core/se/jbee/inject/container/InjectionSite.java#L92

Yields (pun intended):

```text
src\core\se\jbee\inject\container\InjectionSite.java:56: error: invalid use of a restricted identifier 'yield'
				: yield(generators[i], site.instanced(hint.relativeRef));
				  ^
  (to invoke a method called yield, qualify the yield with a receiver or type name)
src\core\se\jbee\inject\container\InjectionSite.java:80: error: invalid use of a restricted identifier 'yield'
					preResolvedArgs[i] = yield(resource,
					                     ^
  (to invoke a method called yield, qualify the yield with a receiver or type name)
2 errors
```

See https://github.com/jbee/silk/pull/77#discussion_r393860959 -- rename it to `generate` as well?

--- a/src/core/se/jbee/inject/container/InjectionSite.java
+++ b/src/core/se/jbee/inject/container/InjectionSite.java
@@ -53,7 +53,7 @@ public final class InjectionSite {
 			Hint<?> hint = hints[i];
 			args[i] = generators[i] == null
 				? injector.resolve(site.instanced(hint.relativeRef))
-				: yield(generators[i], site.instanced(hint.relativeRef));
+				: generate(generators[i], site.instanced(hint.relativeRef));
 		}
 		return args;
 	}
@@ -77,7 +77,7 @@ public final class InjectionSite {
 				Resource<?> resource = injector.resolve(resourceDep);
 				if (resource.scoping.isStableByDesign()) {
 					//TODO and not has type variable involved
-					preResolvedArgs[i] = yield(resource,
+					preResolvedArgs[i] = generate(resource,
 							site.instanced(hint.relativeRef));
 				} else {
 					lazyArgIndexes[lazyArgCount++] = i;
@@ -89,7 +89,7 @@ public final class InjectionSite {
 	}
 
 	@SuppressWarnings("unchecked")
-	private static <I> I yield(Generator<I> gen, Dependency<?> dep) {
+	private static <I> I generate(Generator<I> gen, Dependency<?> dep) {
 		return gen.generate((Dependency<? super I>) dep);
 	}
 }

